src/index.html

```diff
@@ -22,6 +22,495 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header__link"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
+          class="header__logo"
+        />
+      </a>
+      <nav class="header-nav">
+        <ul class="header-nav__list">
+          <li>
+            <a
+              href="#"
+              class="header-nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="header-nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="header-nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              data-qa="nav-hover"
+              class="header-nav__link"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="header-nav__link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="header-nav__link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="header-nav__link"
+            >
+              Photo
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="header-nav__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <div class="grid-item">
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <div class="card__top">
+            <img
+              src="./images/imac.jpeg"
+              class="card__picture"
+              alt="imac-picture"
+            />
+          </div>
+
+          <div class="card__product-info">
+            <h1 class="card__product-name">
+              APPLE A1419 iMac 27" Retina
+              <br />
+              5K Monoblock (MNED2UA/A)
+            </h1>
+
+            <p class="card__product-code">Product code: 195434</p>
+
+            <div class="card__reviev">
```

Comment:
There seems to be a typo in the class name `card__reviev`. According to the English language and consistency with other class names, it should be `card__review`.